pkg/relayer/proxy/relay_meter.go

```diff
@@ -197,14 +197,18 @@ func (rmtr *ProxyRelayMeter) ShouldRateLimit(ctx context.Context, reqMeta servic
 		maxAllowedOverServicing := appRelayMeter.maxCoin.Add(rmtr.overServicingAllowanceCoins)
 		if maxAllowedOverServicing.IsLT(newConsumedCoin) {
 			rmtr.logger.Warn().Msgf(
-				"application has been rate limited, stake needed: %s, has: %s",
+				"application with address %q has been rate limited, stake needed: (%s), has: (%s)",
+				reqMeta.SessionHeader.ApplicationAddress,
 				newConsumedCoin.String(),
-				appRelayMeter.maxCoin.String(),
+				maxAllowedOverServicing.String(),
```

Comment:
This is feature creep, but will be easy for iteration in the coming months.

1. Add a flag called `--allow-overservicing`
2. When the relay meter reaches the max for the session, stop population the SMST
3. Continue servicing requests normally